grease-cli/tests/sanity/pass/test-pointer-offset/test.c

```diff
@@ -0,0 +1,18 @@
+/* Copyright (c) Galois, Inc. 2024 */
+
+// x64: flags {"--symbol", "test", "--no-heuristics", "--initial-precondition", "tests/sanity/pass/test-pointer-offset/init.shape"}
```

Comment:
Optional nit: I kind of like grouping these into conceptual sets. Helps separate the generic ones (`--symbol`) from the ones that are particular to this test, and reduces line length.
```suggestion
// x64: flags {"--symbol", "test"}
// x64: flags {"--no-heuristics", "--initial-precondition", "tests/sanity/pass/test-pointer-offset/init.shape"}
```